sphinx/ext/autodoc/__init__.py

```diff
@@ -1369,16 +1390,20 @@ def import_object(self):
         else:
             # if it's not a data descriptor
             self._datadescriptor = False
+
+        if isabstract(self.object):
+            self.options['abstract'] = True
```

Comment:
Why not in `MethodDocumenter`?

Reply:
Sorry, I didn't pay much mind about the best place, so I put it in the most generic class. Judging by the type hierarchy, I think the `ClassLevelDocumenter` would be the right level (`MethodDocumenter` doesn't sound like it covers properties as well)

Reply:
Let me look over this a bit, I misjudged some things and need to get used to the codebase again.
I'll get back to you shortly.

Reply:
This was the wrong place, I now did this in the `MethodDocumenter` and `AttributeDocumenter`, respectively. :-)

Reply:
Well, it *is* in the `MethodDocumenter`, but here we are in the `AttributeDocumenter`. This is responsible for documenting abstract `@property`s: If you comment this out, the test case fails precisely at the point where it's trying to verify the `:abstract:` flag on `AbstractBase.value`, which is defined as an abstract property using `@abstractmethod` in the usual way.